openquakeplatform/openquakeplatform/static/js/hazus.js

```diff
@@ -39,25 +41,79 @@ var startApp = function() {
     map.panTo(new L.LatLng(38.2, -101.6));
     map.scrollWheelZoom.enable();
 
-    // This layer is used for the visual representation of the data
-    var hazus = L.tileLayer(TS_URL + '/v2/ged-hazus-level1/{z}/{x}/{y}.png');
-    layerControl.addOverlay(hazus, "Hazus Level 1 Building Fractions");
-    map.addLayer(hazus);
+    $('#map-tools').prepend('<select id="external-layers-menu">'+
+            '<option>Select layers</option>'+
+            '<option value="1">Hazus Level 1 Building Fractions</option>'+
+            '<option value="2">US Counties</option>'+
+            '<option value="3">PAGER Dwelling Fractions Level 0'+
+        '</select>'
+    );
 
```

Comment:
the approach "unconditional html injection via js" seems to me not so clean.
could you try to redefine this block of code
```
{% block map_tools %}
    {% include "base_map_options.html" %}
{% endblock %}
```

 into ```./openquakeplatform/openquakeplatform/templates/hazus.html```
and add into it this code directly ? 
```html
<select id="external-layers-menu">
  <option>Select layers</option>
  <option value="1">Hazus Level 1 Building Fractions</option>
  <option value="2">US Counties</option>
  <option value="3">PAGER Dwelling Fractions Level 0</option>
</select>
```